README.md

```diff
@@ -39,6 +39,46 @@ Our product will allow creating, editing and tracking of bike components.
 - MySQL
 
 ## Running the project
-To run the project locally, make sure to have Docker set up following the [documentation](https://github.com/nasaku898/soen390-team08/wiki/Setting-up-Docker).
 
-Once done, simply write `docker-compose up --build` in your terminal. The project will be available at http://localhost:3000.
+### Locally
+1. In the file `/bike-erp/backend/src/main/config/config.ts`, make sure that the local database variables are uncommented and that the external database variables are commented. You have to enter the password (and other information if necessary) of your local mysql database in the config.ts file.
```

Comment:
I feel like we should have a `Setting up a Database` page in the wiki that explains how to set up a local MySQL database that we can then reference here as a preliminary step to running the project locally.